Library/Homebrew/service.rb

```diff
@@ -355,6 +355,20 @@ def command
       @run&.map(&:to_s)
     end
 
+    sig { params(value: T.nilable(Integer)).returns(T.nilable(Integer)) }
```

Comment:
Why/when would you want to pass in a `value: nil`?
```suggestion
    sig { params(value: Integer).returns(T.nilable(Integer)) }
```
may make more sense if not

Reply:
Correct, we can remove nil but I was following the format of the rest of parameters like restart_delay which is also defined nilable
https://github.com/Homebrew/brew/blob/0d70ca7ab0d9a669e85e50bcfedca7cc9c7135c3/Library/Homebrew/service.rb#L211

Reply:
CC @SMillerDev for thoughts here, I think it'd probably make sense to make none of these nilable.

Reply:
Most of these methods simply return the current value when `nil` (i.e. no argument) is passed. We don't seem to be calling any of these without an argument though.

Reply:
@dduugg @reitermarkus Also would welcome your thoughts on how, in a future PR, we could refactor this type handling to lean more on Sorbet (bearing in mind that e.g. formulae will almost never be evaluated at runtime to use Sorbet and are currently never type checked by Sorbet)

Reply:
I don't think i have any especially creative suggestions. You can try to bump these files to `typed: strict` to catch more edge cases. I believe we do runtime typechecking in tests, so covering every `service` DSL in tests could do the trick. A rubocop that flags unacceptable`nice` values could help too (it's tricky if you're allowing vars to be passed to `nice`, but it could enforce that `nice` is followed by a literal `Integer` in the required range).

Reply:
Thanks @dduugg!

Reply:
> CC @SMillerDev for thoughts here, I think it'd probably make sense to make none of these nilable.

Not a clue, pretty sure I just copied this from livecheck